lib/fsmx.ex

```diff
@@ -2,29 +2,39 @@ defmodule Fsmx do
   @moduledoc """
   """
 
+  alias Fsmx.Fsm
+
   @type state_t :: binary() | atom()
 
   @spec transition(struct(), state_t()) :: {:ok, struct} | {:error, any}
```

Comment:
i think these `@spec`s will need to be updated, too

Reply:
Yes, missed those.